src/components/Collapsible/Collapsible.scss

```diff
@@ -1,33 +1,30 @@
 @import '../../styles/common';
 
 .Collapsible {
-  overflow: hidden;
-  max-height: 0;
   padding-top: 0;
   padding-bottom: 0;
-  opacity: 0;
-  will-change: opacity, max-height;
-}
-
-.animating {
-  transition-property: opacity, max-height;
-  transition-duration: duration(slow);
+  height: 0;
+  overflow: hidden;
+  will-change: height;
+  transition-property: height;
+  transition-duration: duration(fast);
```

Comment:
Do we want to use `var(--p-duration-1-0-0)` here instead? 

Edit: nvm looks like we need to use tokens to match some JS below